docs/LearnCalico.md

```diff
@@ -0,0 +1,358 @@
+# Calico networking in a containerized environment
+
+This GitHub repository is the entry point for running a Calico network in a
+containerized environment.
+
+The `calico-docker` repository allows users to easily configure endpoints and 
+network policy with the `calicoctl` command line tool. `calico-docker` utilizes 
+the Calico Felix process from within a container to interface with a host's 
+Linux Kernel, providing configurable connectivity between containers in a 
+network and the outside world.
+
+
+The repository provides:
+
+- A `calico/node` docker image that is used to run the Calico Felix agent 
+  (responsible for IP routing and ACL programming) and a BGP agent (BIRD) used
+  for distributing routes between hosts.
+- A command line tool `calicoctl` used for starting a `calico/node` container,
+  configuring Calico policy, adding and removing containers in a Calico network via 
+  orchestration tools, and managing certain network and diagnostic administration.
+
+A number of additional GitHub repositories provide library functionality and
+orchestration tools that are utilized by calicoctl.  These are listed below:
+
+ - [calico](https://github.com/projectcalico/calico): Implements the Felix 
+   process that interfaces with the Linux Kernel to 
+   configure routes and ACLs that control network policy connectivity.  Felix 
+   runs as a process within the `calico/node` container.
+
+
+ - [libcalico](https://github.com/projectcalico/libcalico): Contains code for 
+   assigning IP addresses to endpoints, interfacing with 
+   Linux namespaces, and storing data in the etcd datastore.  The libcalico 
+   library lives within `calicoctl`.
+
+
+ - [calico-kubernetes](https://github.com/projectcalico/calico-kubernetes): 
+   Implements the Calico plugin for running Calico with kubernetes 
+   orchestrator.  This is used when the Calico node is started with the 
+   `--kubernetes` flag.
+
+
+ - [calico-mesos](https://github.com/projectcalico/calico-mesos): Implements 
+   the Calico plugin for running Calico with the [mesos] orchestrator.
+
+
+ - [calico-rkt](https://github.com/projectcalico/calico-rkt): Implements the 
+   Calico plugin for running Calico with the [rkt] orchestrator.
+
+
+ - [libnetwork-plugin](https://github.com/projectcalico/libnetwork-plugin): 
+   Implements Calico plugin support for the Docker libnetwork 
+   networking plugin.  This is used when the Calico node is started with the 
+   `--libnetwork` flag
+
+
+## Minimum Requirements
```

Comment:
I'd rename to "Components of a Basic Calico Deployments", or something along those lines